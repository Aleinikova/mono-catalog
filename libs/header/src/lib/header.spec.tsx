import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';

import Header from './header';

describe('Header', () => {
  it('should render successfully', () => {
    const { baseElement, getByText } = render(
      <Header Logo={() => <div>logo</div>} />
    );
    expect(baseElement).toBeTruthy();
    expect(getByText('logo')).toBeInTheDocument();
  });
  it('should match snapshot', async () => {
    const tree = renderer
      .create(<Header Logo={() => <div>logo</div>} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
