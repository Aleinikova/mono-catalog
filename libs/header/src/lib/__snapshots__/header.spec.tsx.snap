// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<header
  className="___quniqg0_1a7ai4t f22iagw f1869bpl f1ggmyuv fsjp82u fz66xbh f1nochiw fxugw4r"
>
  <div>
    logo
  </div>
</header>
`;
